src/tools/bin/tbg

```diff
@@ -374,12 +374,29 @@ if [ ! -f "$cfg_file" ] ; then
 fi
 
 # cfg file sanity check - space after \ at EOL ?
-cfg_err=`egrep "\\\\\[[:space:]]+$" $cfg_file | wc -l`
+cfg_err=`egrep "\\\\\[[:blank:]]+$" $cfg_file | wc -l`
 if [ $cfg_err != 0 ] ; then
     echo "ERROR: file \"$cfg_file\" contains spaces after line continuation \\"
     echo "Check the following lines for end-of-line spaces:"
     echo ""
     egrep -n "\\\[[:space:]]+$" $cfg_file
```

Comment:
Fixed it: https://github.com/ComputationalRadiationPhysics/picongpu/pull/1777/commits/4043dce648af730aa21e177c9ec0bb4dd51fdd80